### DISCUSSION TITLE
Deployed two different branches, one for production and the other for development?

### DISCUSSION THREAD

**Discussion by BaziotaBeans** [vercel/next.js#43296] [Help]
Have you already deployed two different branches, one for production and the other for development?
I have a Next.js project already in production, I created a new branch for development, I would like to know how to implement it in vercel too

**Comment by icyJoseph** [comment]
In Vercel, you just push your branch to the remote, and it'll make a preview of that commit HEAD. When you push to the "release" branch, the main deployment is updated.

**Reply by BaziotaBeans** [reply]
But that won't make me have two implementations at the same time, I want something that allows me to have a test version in the air and a production version, thank you very much for the reply.

**Reply by icyJoseph** [reply]
I'm sorry I don't understand how is that not the same? In various projects I have a dev branch and a different release branch, so that I have many versions as you say.

**Reply by BaziotaBeans** [reply]
Sorry for the confusion, thank you very much that's exactly what you said...

**Reply by icyJoseph** [reply]
https://vercel.com/docs/concepts/deployments/preview-deployments. You can read more about these here:) hope that helps!

**Reply by BaziotaBeans** [reply]
Super thanks
